tests/components/homekit_controller/test_binary_sensor.py

```diff
@@ -71,4 +81,4 @@ def create_smoke_sensor_service(accessory):
     state = await helper.poll_and_get_state()
     assert state.state == "on"
 
-    assert state.attributes["device_class"] == "smoke"
+    assert state.attributes["device_class"] == DEVICE_CLASS_SMOKE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5069901)